src/checkTargetMatchToPR.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+const semverDiff = require('semver/functions/diff')
+
+const { targetOptions, semanticVersionOrder } = require('./getTargetInput')
+
+const expression = /from ([^\s]+) to (.+)$/
+
+const checkTargetMatchToPR = (prTitle, target) => {
+  if (target === targetOptions.major) {
+    return true
+  }
+  const match = expression.exec(prTitle)
+
+  if (match === null) {
```

Comment:
let's not check against null, let's simply test a falsy value for more robustness